packages/blockchain-wallet-v4-frontend/src/components/Navbar/MobileDropdown/template.tsx

```diff
@@ -62,12 +63,16 @@ const MobileDropdown = () => {
 
   return (
     <NavbarButton data-e2e='mobileQrCode' ref={ref}>
-      <Icon color={colors.grey400} name={IconName.PHONE} size='sm' onClick={handleMenuToggle} />
+      {/* @ts-ignore */}
+      <Icon label='phone' size='sm' color='grey400' onClick={handleMenuToggle}>
+        <IconPhone />
+      </Icon>
```

Comment:
@pedroapfilho @awiermanncasas-bc 
1) Icon component and TS dont like the `onClick` attribute. can we fix?